internal/cmd/password/create.go

```diff
@@ -76,7 +85,7 @@ func CreateCmd(ch *cmdutil.Helper) *cobra.Command {
 		},
 	}
 	cmd.PersistentFlags().StringVar(&flags.role, "role",
-		"admin", "Role defines the access level, allowed values are : reader, writer, readwriter, admin. By default it is admin.")
+		"", "Role defines the access level, allowed values are : reader, writer, readwriter, admin. By default it is admin.")
```

Comment:
Can we update the copy here?

```suggestion
		"", "Role defines the access level, allowed values are : reader, writer, readwriter, admin. By default it is admin.")
```
```suggestion
		"", "Role defines the access level, allowed values are : reader, writer, readwriter, admin. Defaults to 'reader' for replicas, otherwise defaults to 'admin'.")
```